css/style.css

```diff
@@ -1,96 +1,355 @@
+    color: rgb(255, 255, 255);
+    box-sizing: border-box;
+    width: 100%;
+    background: #3c3a39;
+  }
 
-.fa-bars::before {
-  color: white;
-  text-decoration: none;
-}
+  header .logo {
+    color: #fff;
+    text-decoration: none;
+  }
 
-.Home-content {
-  display: flex;
-  box-sizing: border-box;
-  flex-direction: column;
-  align-items: flex-start;
-  padding: 120px 24px;
-  gap: 12px;
-  width: 100%;
-  top: 92px;
-  background: #1c1a19;
-  background-image: url(/assets/images/vec.png);
-  background-position: bottom right;
-  background-repeat: no-repeat;
-}
+  .fa-bars::before {
+    color: white;
+    text-decoration: none;
+  }
 
-h1 {
-  font-family: Arial, Helvetica, sans-serif;
-  font-weight: 400;
-  font-size: 56px;
-  line-height: 64px;
-  letter-spacing: 0.3px;
-  color: #ff6b00;
-}
+  .home-content {
+    display: flex;
+    box-sizing: border-box;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 120px 24px;
+    gap: 12px;
+    width: 100%;
+    top: 92px;
+    background: #1c1a19;
+    background-image: url(/assets/images/vec.png);
+    background-position: bottom right;
+    background-repeat: no-repeat;
+  }
 
-h2 {
-  width: 327px;
-  height: 27px;
-  font-family: Arial, Helvetica, sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 24px;
-  line-height: 32px;
-  display: flex;
-  align-items: center;
-  color: #fff;
-}
+  h1 {
+    font-family: 'Arial, Helvetica', sans-serif;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.3px;
+    color: #ff6b00;
+  }
 
-p {
-  width: 327px;
-  height: 123px;
-  font-family: Arial, Helvetica, sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  color: #fff;
-  flex: none;
-  flex-grow: 0;
-}
+  h2 {
+    width: 327px;
+    height: 27px;
+    font-family: 'Arial, Helvetica', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
 
-.menu {
-  display: flex;
-  flex-flow: nowrap;
-  list-style: none;
-  justify-content: space-between;
-}
+  p {
+    width: 327px;
+    height: 123px;
+    font-family: ' Arial, Helvetica', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    flex: none;
+    flex-grow: 0;
+  }
+
+  .menu {
+    display: flex;
+    flex-flow: nowrap;
+    list-style: none;
+    justify-content: space-between;
+  }
+
+  .social {
+    list-style: none;
+    display: flex;
+    justify-content: flex-start;
+    gap: 10px;
+    vertical-align: bottom;
+    line-height: 1.22;
+    width: 200px;
+  }
+
+  hr {
+    width: 327px;
+    height: 0;
+    border: 1px solid #212121;
+    flex: none;
+    flex-grow: 0;
+  }
+
+  .recent-works {
+    width: 100%;
+    margin: 0 24px;
+  }
+
+  .recent-works h1 {
+    font-family: 'Crete Round', sans-serif;
```

Comment:
Please use the good font to stick to Figma

<img width="784" alt="image" src="https://user-images.githubusercontent.com/7243132/170840651-d717500a-6a5c-455f-a345-a3ecc49934e8.png">